### PR TITLE
SCP-4377: Remove non playground packages

### PR DESCRIPTION
**Pull request by hrajchert**
This is the first PR on this repository. The repo was cloned from [marlowe-cardano](https://github.com/input-output-hk/marlowe-cardano), and this PR prunes all packages that are not relevant to building the Marlowe Playground.

Future Work:
* SCP-4378: Upgrade PlayGround to PureScript 0.15 (this is needed by task below)
* SCP-4379: Remove marlowe-web-commons in favour of [purescript-marlowe](https://github.com/input-output-hk/purescript-marlowe)
* SCP-4383: Redo nix structure on the new marlowe-playground

Pre-submit checklist:
- Branch
    - [X] Tests are provided (if possible)
    - [X] Commit sequence broadly makes sense
    - [X] Key commits have useful messages
    - [X] Relevant tickets are mentioned in commit messages
    - [X] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [X] Self-reviewed the diff
    - [X] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -59,6 +53,15 @@ package cardano-wallet-launcher
 package cardano-wallet-core-integration
   optimization: False
 
+source-repository-package
```
Comment (jhbertra) [inline cabal.project:56]: There are probably quite a few Haskell packages we can prune from here

File: bitte/default.nix [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{ marlowe-playground, marlowe-pab, web-ghc, marlowe-dashboard, marlowe-dashboard-server, cardano-node, cardano-wallet, plutus-chain-index, docs, pkgs, sources }:
+{ marlowe-playground, web-ghc, docs, pkgs, sources }:
 let
   staticSite = pkgs.callPackage (sources.plutus-apps + "/bitte/static-site.nix") { };
   playgroundStatic = pkgs.callPackage (sources.plutus-apps + "/bitte/playground-static.nix") { inherit staticSite; docs = docs.site; };
```
Comment (shlevy) [inline bitte/default.nix:4]: We don't need `wait-for-socket` for the playground

File: bitte/default.nix [hunk 3]
```diff
@@ -80,4 +52,4 @@ in
   node-socat = pkgs.callPackage ./node-socat.nix {
```
Comment (shlevy) [inline bitte/default.nix]: We don't need `node-socat`

File: cabal.project [hunk 4]
```diff
@@ -1,14 +1,8 @@
...
 index-state: 2022-02-22T20:47:03Z
 
-packages: marlowe
-          marlowe-actus
-          marlowe-playground-server
-          marlowe-dashboard-server
-          marlowe-symbolic
-          marlowe-cli
+packages: marlowe-playground-server
           marlowe-contracts
```
Comment (shlevy) [inline cabal.project:5]: Why is this here? Shouldn't it stay in marlowe-cardano?

File: cabal.project [hunk 5]
```diff
@@ -59,6 +53,15 @@ package cardano-wallet-launcher
...
 
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/marlowe-cardano
+  tag: f5a0f64d8697dd7a29af91b11530039dc98a1d44
+  subdir:
+    marlowe
+    marlowe-symbolic
+    marlowe-actus
+
```
Comment (shlevy) [inline cabal.project:64]: If you have the patience, I might just delete all of the other `source-repository-package` entries and only add back in the ones needed to build.

File: default.nix [hunk 6]
```diff
@@ -93,7 +63,6 @@ rec {
```
Comment (shlevy) [inline default.nix]: Can remove this, and also `actus-tests` from the flake.nix.

**Review by shlevy — APPROVED** [review]
Comments include options for more removals, looks good though.

File: cabal.project (hunk 1 above)
Comment (hrajchert) [reply cabal.project:56]: Yes I know, but I wanted to leave that for a second step, so I can first unblock PR's that required the Playground to be in its own repository

File: cabal.project (hunk 4 above)
Comment (hrajchert) [reply cabal.project:5]: As I understand, those are here because the PlayGround builds them and use them as part of the example programs. But I'll see if we can get rid of it in the future.

File: cabal.project (hunk 5 above)
Comment (hrajchert) [reply cabal.project:64]: Yes, I wanted to do that as a second step
